packages/@atjson/document/src/index.ts

```diff
@@ -375,6 +375,34 @@ export default class Document {
     return matches;
   }
 
+  canonical() {
+    let canonicalDoc = this.clone();
+    canonicalDoc.where({type: '-atjson-parse-token'}).update(a => {
```

Comment:
```suggestion
    canonicalDoc.where({ type: '-atjson-parse-token' }).update(a => {
```

Adding some spaces :)